### ISSUE TITLE
Consecutive lists could change of style

### ISSUE THREAD

**Issue by Sylry** [thephpleague/html-to-markdown#153]
If you have in your HTML source 2 consecutive lists, like this:
```html
<ul>
  <li>item of first list</li>
  <li>item of first list</li>
</ul>
<ul>
  <li>item of second list</li>
</ul>
```
Actual code generates this:
```
- item of first list
- item of first list

- item of second list
```
instead of that:
```
- item of first list
- item of first list

* item of second list
```

I propose to add a config to make it possible to change the style of the list between each list.
If you use this config, consequence is that even if there is a paragraph between 2 lists, the style is changed.

PR will follow

**Comment by Sylry** [comment]
Thanks for the response, will try different approach, without #152

**Comment by Sylry** [comment]
I think this one is more like you want and it does not need #152: #155

**Comment by colinodell** [comment]
Implemented via #155
